index.html

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I'd prefer our site be written in markdown if at all possible.

FYI you can use some HTML inside of the markdown, but it looks like everything you use here is supported natively.